docs/guides/runbooks_steps_google_docs.md

```diff
@@ -0,0 +1,66 @@
+subcategory: "Runbooks"
+---
+
+# Google Docs
+
+~> **Note** You must have the Google Docs integration installed in FireHydrant
+for any Google Docs runbook steps to work properly.
+
+The FireHydrant Google Docs integration allows FireHydrant users to export retrospective
+reports to Google Docs spaces after they are published.
+
+### Available Steps
+
+* [Export Retrospective](#export-retrospective)
+
+## Export Retrospective
+
+The [Google Docs **Export Retrospective** step](https://support.firehydrant.com/hc/en-us/articles/5502417924116-Exporting-Retrospectives-to-Google-Docs)
+allows FireHydrant users to export retrospective reports to Google Docs spaces after they are published.
+
+### Export Retrospective - Example Usage
+
+Basic usage:
+```hcl
+data "firehydrant_runbook_action" "google_docs_export_retro" {
+  integration_slug = "google_docs"
+  slug             = "export_retrospective"
+  type             = "incident"
+}
+
+resource "firehydrant_runbook" "google_docs_export_retro_runbook" {
+  name = "google-doc-export-retro-runbook"
+
+  steps {
+    name      = "Export Retrospective to Google Docs"
+    action_id = data.firehydrant_runbook_action.google_docs_export_retro.id
+
+    config = jsonencode({
+      title_template = "Retrospective: {{ retro.name }}"
+      body_template  = "# Retrospective: {{ retro.name }}\n\n**Incident:** [{{ incident.severity }} - {{ incident.name}}]({{ incident.incident_url }}) ({{ retro.incident_active_duration }})\n\n{{ retro.summary }}\n\n{%- if incident.customer_impact_summary != blank or retro.impacts != empty -%}\n### Impact\n{% if incident.customer_impact_summary != blank -%}\n  {{ incident.customer_impact_summary }}\n{% endif %}\n\n\n{% if retro.impacts != empty %}\n| Type | Name | Condition |\n|------|------|-----------|\n{%- for impact in retro.impacts %}\n| {{ impact.type }} | {{ impact.name }} | {{ impact.condition }} |\n{%- endfor %}\n{% endif %}\n{% endif %}\n\n\n### Milestones\n\n| Milestone | Started | Duration |\n|-----------|---------|----------|\n{%- for milestone in retro.milestones %}\n| {{ milestone.type }} | {{ milestone.occurred_at }} | {{ milestone.duration }} |\n{%- endfor %}\n\n\n{% if retro.incident_roles != empty %}\n### Responders\n\n| Role | User |\n|------|------|\n{%- for role in retro.incident_roles %}\n| {{ role.name }} | {{ role.user }} |\n{%- endfor %}\n{% endif %}\n\n\n## Analysis\n\n{% if retro.contributing_factors != empty -%}\n### Contributing Factors\n{% for factor in retro.contributing_factors %}\n- {{ factor.summary }}\n{%- endfor %}\n{%- endif %}\n\n\n{% if retro.questions != empty -%}\n### Questions\n\n{% for question in retro.questions %}\n**{{ question.title }}**\n\n{{ question.body }}\n{% endfor %}\n{% endif %}\n\n\n---\n\n## Important events\n\n{% if retro.starred_events != empty %}\n{% for event in retro.starred_events %}\n_{{ event.occurred_at}}_ ({{event.created_by}})\n\n{{event.body}}\n{% endfor %}\n{% endif %}\n"
+    })
+
+    automatic = true
```

Comment:
change this to false and warn about the fact that this runbook step won't work until your enter retro started
```suggestion
    automatic = false
```